services/121-service/src/config.ts

```diff
@@ -5,7 +5,9 @@ export const SUBDOMAIN = process.env.NODE_ENV == 'production' ? '121-service/' :
 
 const tyknIMS = 'http://11.0.0.3:50001/api/';
 const orgIMS = 'http://11.0.0.4:50002/api/';
-const userIMS = 'http://11.0.0.5:50003/api/';
+
+export const PRODUCTION_URL = 'https://production-vm.121.global/121-service/'
+export const STAGING_URL = 'https://test-vm.121.global/121-service/';
```

Comment:
Similar to the use of `config.ts` in the PA-accounts-service, these URLS don't need to be hard-coded/committed.

A single `APP_URL`/`SERVICE_URL`/`PUBLIC_URL`/`EXTERNAL_URL` or `DEPLOYMENT_URL` could be set here based on an ENV-variable.

```ts
export const PUBLIC_URL = process.env.PUBLIC_URL || 'http://localhost:3000/api/'; // That last default-value would be save, but can be omitted...
```